lib/happo/server.rb

```diff
@@ -57,40 +60,46 @@ def h(text)
         {
           description: '<First> with "test"',
           viewport: 'small',
-          url: 'http://placehold.it/350x150',
+          previous: 'http://placehold.it/350x150',
+          diff: 'http://placehold.it/350x150',
+          current: 'http://placehold.it/350x150',
```

Comment:
Yeah, I kind of wanted to switch these out for "real" diffs at some point too. I think we could use some of the diffs produced by the brigade ci run.